cmd/cmd_test.go

```diff
@@ -662,3 +664,24 @@ func TestBadCaseIsJustWarning(t *testing.T) {
 		t.Errorf("expect no error (just a warning), got %q", err)
 	}
 }
+
+func TestSetupHappensBeforeOtherCommands(t *testing.T) {
```

Comment:
Hey @dustmop do you agree with this location as the right spot for this test? I read the issue as alluding to us including this as a part of `setup_test.go` but that felt a little off to me because we're testing this for any command that's not `qri setup`. I also didn't love the idea of popping a similar test in each of the `_test.go` files for the respective qri commands since that was super duplicative, so I settled on this file which seemed to encompass tests which span different commands? Let me know if I'm wrong about its intent, though

Reply:
I think this is a good spot for it. My initial instance was also perhaps `setup_test.go`, but you're right that this is testing non-setup commands, so this generic file is a better location.